tests/boards/mec15xxevb_assy6853/i2c_mainline/src/main.c

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2016 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <drivers/i2c.h>
+#include <zephyr.h>
+#include <ztest.h>
+
+#define I2C_DEV_NAME DT_LABEL(DT_ALIAS(i2c1))
+
+/* command to configure port direction of NXP PCA95xx */
+#define REG_CONF_PORT0 0x06U
+
+/* test data used to write into registers */
+uint8_t test_data[2] = {0xAA, 0xAA};
+
+/**
+ * @brief Test i2c write and read with pca95xx
+ * @details
+ * - get i2c mainline device
+ * - write data into pca95xx
+ * - read data from pca95xx
+ * - check read data whether correct
+ */
+void test_i2c_mainline(void)
```

Comment:
I think we should remove "mainline" in the function name.

Reply:
updated